src/app/components/source/UserComponent.js

```diff
@@ -1,122 +1,41 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { browserHistory } from 'react-router';
-import { FormattedMessage } from 'react-intl';
-import Tabs from '@material-ui/core/Tabs';
-import Tab from '@material-ui/core/Tab';
 import cx from 'classnames/bind';
 import UserEmail from '../user/UserEmail';
 import UserInfo from './UserInfo';
-import UserPrivacy from './UserPrivacy';
-import UserSecurity from './UserSecurity';
-import UserInfoEdit from './UserInfoEdit';
-import { can } from '../Can';
 import PageTitle from '../PageTitle';
 import CheckContext from '../../CheckContext';
-import SwitchTeamsComponent from '../team/SwitchTeamsComponent';
 import styles from './User.module.css';
 
 class UserComponent extends React.Component {
-  constructor(props) {
-    super(props);
-    const { tab } = this.props.params;
-    const showTab = (typeof tab === 'undefined') ? 'workspaces' : tab;
-    this.state = {
-      showTab,
-    };
-  }
-
   componentWillMount() {
     const { user } = this.props;
     if (!user.is_active) {
       browserHistory.push('/check/not-found');
     }
+    if (user.dbid === this.getContext().currentUser.dbid) {
+      browserHistory.push('/check/me');
+    }
   }
 
   getContext() {
     return new CheckContext(this).getContextStore();
   }
 
-  handleTabChange = (e, value) => {
-    browserHistory.push(`/check/user/${this.props.user.dbid}/${value}`);
-    this.setState({
-      showTab: value,
-    });
-  };
-
   render() {
     const { user } = this.props;
-    const isEditing = this.props.route.isEditing && can(user.permissions, 'update User');
     const context = this.getContext();
-    const isUserSelf = (user.id === context.currentUser.id);
+    // const isUserSelf = (user.id === context.currentUser.id);
```

Comment:
Suggestion: delete this line, or add some text explaining why it's a comment.

Reply:
Deleted